chsdi/templates/htmlpopup/cadastralwebmap_opendata.mako

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
```

Comment:
An include is good as well at least we don't have to change these URLs in 2 places